setup.py

```diff
@@ -20,7 +36,6 @@
         ],
         "alembic": ["alembic~=1.4"],
     },
-    license="Apache",
```

Comment:
No longer apache?
This repo it's still open source. No?

Reply:
duplicate, there is already a license field above.